ccmlib/dse_node.py

```diff
@@ -377,7 +375,7 @@ def __generate_server_xml(self):
         with open(server_xml, 'w+') as f:
             f.write('<Server port="8005" shutdown="SHUTDOWN">\n')
             f.write('  <Service name="Solr">\n')
-            f.write('    <Connector port="8983" address="%s" protocol="HTTP/1.1" connectionTimeout="20000" maxThreads = "200" URIEncoding="UTF-8"/>\n' % self.network_interfaces['thrift'][0])
+            f.write('    <Connector port="8983" address="%s" protocol="HTTP/1.1" connectionTimeout="20000" maxThreads = "200" URIEncoding="UTF-8"/>\n' % self.ip_addr)
```

Comment:
This is the main issue for me, if thrift is not enabled and I attempt to use the solr workload, I get an error:

```
  File "/.../python2.7/site-packages/ccmlib/dse_node.py", line 380, in __generate_server_xml
    f.write('    <Connector port="8983" address="%s" protocol="HTTP/1.1" connectionTimeout="20000" maxThreads = "200" URIEncoding="UTF-8"/>\n' % self.network_interfaces['thrift'][0])
TypeError: 'NoneType' object has no attribute '__getitem__'
```